main.css

```diff
@@ -205,3 +205,87 @@ img {
   color: #344563;
   margin-top: 60px;
 }
+
+.footer {
+  background-color: #6070ff;
+  min-height: 779px;
+  max-width: 375px;
+  border-radius: 80px 0 0 0;
+  background-image: url(./images/background.png);
+}
+
+.footer-div h2 {
+  font-family: Poppins, sans-serif;
+  font-size: 40px;
+  font-style: normal;
+  font-weight: 700;
+  line-height: 52px;
+  letter-spacing: 0;
+  text-align: center;
+  color: #fff;
+  padding-top: 103px;
+}
+
+.footer-div p {
+  font-family: Poppins, sans-serif;
+  font-size: 20px;
+  font-style: normal;
+  font-weight: 400;
+  line-height: 28px;
+  text-align: center;
+  color: #fff;
+  width: 375px;
+  padding: 0 24px;
+  margin-top: 12px;
+}
+
+.footer-div .form-style {
+  padding: 0 24px;
+  display: table-caption;
+  margin-top: 87px;
+}
+
+.form-style input {
+  width: 327px;
+  height: 48px;
+  border: 1px solid #cfd8dc;
+  border-radius: 8px;
+  margin-bottom: 20px;
+  padding: 15px 38px 15px 16px;
+  font-family: poppins, sans-serif;
+  font-size: 17px;
+  font-style: normal;
+  font-weight: 400;
+  line-height: 20px;
+  letter-spacing: 0;
+  text-align: left;
+}
+
+.form-style textarea {
+  width: 327px;
+  height: 140px;
+  border: 1px solid #cfd8dc;
+  border-radius: 8px;
+  margin-bottom: 22px;
+  padding: 15px 38px 15px 16px;
+  font-family: Poppins, sans-serif;
+}
+
+.form-style button {
+  padding: 12px 16px;
+  width: 140px;
+  height: 48px;
+  background: #fff;
+  border-radius: 8px;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 17px;
+  text-align: center;
+  letter-spacing: 0.03em;
+  color: #6070ff;
+}
+
+.form-style button:hover {
+  background-color: #6070ff;
+  color: #fff;
+}
```

Comment:
- [ ] Kindly use the flex box in the contact section to layout elements as per requiremnets.